src/librsync.h

```diff
@@ -37,9 +37,36 @@
 #define _RSYNC_H
 
 #include <sys/types.h>
-#include "types.h"
 #include "librsync-config.h"
 
+#define HAVE_UINT64 1 /* assume success, but might be undefined below */
+
+#if defined(HAVE_STDINT_H)
+#  include <stdint.h>
+#else
+# if SIZEOF_UNSIGNED_INT == 4
```

Comment:
stdint is apparently from C99, and generally I think it's reasonable we rely on C99.  And, as you say, blake2.h assumes it.

I'll put up an alternative fix.